api/api/controllers/kvdb_controller.py

```diff
@@ -0,0 +1,210 @@
+    """List KVDB resources.
+
+    Parameters
+    ----------
+    pretty : bool
+        Show results in human-readable format.
+    wait_for_complete : bool
+        Disable timeout response.
+    offset : int
+        First element to return in the collection.
+    limit : int
+        Maximum number of elements to return.
+    select : list
+        Select which fields to return (separated by comma).
+    sort : str
+        Sort the collection by a field or fields (separated by comma). Use +/- at the beginning
+        to list in ascending or descending order.
+    search : str
+        Look for elements with the specified string.
+    q : str
+        Query to filter results by.
+    distinct : bool
+        Look for distinct values.
+    type_ : str
+        Policy type. Allowed values: 'testing' | 'production'.
+    kvdb_id : list[str]
+        Filter by KVDB IDs (e.g. kvdb_id=foo&kvdb_id=bar).
+
+    Returns
+    -------
+    ConnexionResponse
+        API response.
+    """
+    f_kwargs = {
+        'policy_type': type_,
+        'ids': kvdb_id if kvdb_id else None,
+        'offset': offset,
+        'limit': limit,
+        'select': select,
+        'sort_by': parse_api_param(sort, 'sort')['fields'] if sort is not None else ['id'],
+        'sort_ascending': True if sort is None or parse_api_param(sort, 'sort')['order'] == 'asc' else False,
+        'search_text': parse_api_param(search, 'search')['value'] if search is not None else None,
+        'complementary_search': parse_api_param(search, 'search')['negation'] if search is not None else None,
+        'search_in_fields': ['id', 'name', 'integration_id'],
+        'q': q,
+        'distinct': distinct
+    }
+
+    dapi = DistributedAPI(
+        f=kvdb.list_kvdbs,
+        f_kwargs=remove_nones_to_dict(f_kwargs),
+        request_type='local_master',
+        is_async=True,
+        wait_for_complete=wait_for_complete,
+        logger=logger,
+        rbac_permissions=request.context['token_info']['rbac_policies']
+    )
+    data = raise_if_exc(await dapi.distribute_function())
+
+    return json_response(data, pretty=pretty)
+
+
+async def post_kvdbs(body: dict, pretty: bool = False, wait_for_complete: bool = False,
+                     type_: str = None) -> ConnexionResponse:
+    """Create a KVDB.
+
+    Parameters
+    ----------
+    body : dict
+        JSON body with the KVDB item to create. Expected fields:
+          - id: str
+          - integration_id: str (optional)
+          - name: str
+          - content: object (K/V map)
+    pretty : bool
+        Show results in human-readable format.
+    wait_for_complete : bool
+        Disable timeout response.
+    type_ : str
+        Policy type. Allowed values: 'testing' | 'production'.
+
+    Returns
+    -------
+    ConnexionResponse
+        API response.
+    """
+    f_kwargs = {
+        'policy_type': type_,
+        'item': body or {}
+    }
+    
+    KVDBCreateModel.from_dict(body or {})
```

Comment:
If the body is empty it will be `{}` because it can't be `None` (it's required)

```suggestion
        'item': body
    }
    
    KVDBCreateModel.from_dict(body)
```